<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitBranchDialog</class>
 <widget class="QDialog" name="GitBranchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Branch/Switch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Switch To</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbBranch">
        <property name="text">
         <string>Branch</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="lstBranches">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="chkForce">
        <property name="text">
         <string>Overwrite working tree changed(force)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Track</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="rbForceTrack">
           <property name="toolTip">
            <string>Pass --track to git</string>
           </property>
           <property name="text">
            <string>Force Track</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbForceNoTrack">
           <property name="toolTip">
            <string>Pass --no-track to git</string>
           </property>
           <property name="text">
            <string>Force No Track</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbNonSpecifyTrack">
           <property name="toolTip">
            <string>Neither --track nor --no-track is passed to git</string>
           </property>
           <property name="text">
            <string>Not Specifiied</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtNewBranch"/>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkCreate">
        <property name="text">
         <string>Create New Branch</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="chkMerge">
        <property name="toolTip">
         <string>Merge between original branch, working tree contents and the branch to switch to</string>
        </property>
        <property name="text">
         <string>Merge</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="chkForceCreation">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Force Creation (Reset branch if exists)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
